sale_order_incoterm_place/models/sale_order.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    destination_place = fields.Char("Place of Destination")
```

Comment:
```suggestion
    incoterm_place = fields.Char()
```
I'd name it incoterm_place.  This place is not necessarily the destination. It can be a named port or seller's premise, for example, depending on the incoterm.